instance/models/openedx_instance.py

```diff
@@ -117,12 +118,17 @@ def get_load_balancer_configuration(self, triggered_by_instance=False):
         for appserver in active_appservers:
             server_name = "appserver-{}".format(appserver.pk)
             ip_address = appserver.server.public_ip
-            if ip_address:
-                appserver_vars.append(dict(ip_address=ip_address, name=server_name))
-            else:
+            if not ip_address:
                 appserver.logger.error(
                     "Active appserver does not have a public IP address. This should not happen."
+                    "Updating internal status to double check."
```

Comment:
We either need a space at the end of the previous line, or in front of this one. Otherwise they'll be concatenated awkwardly.